hadoop-connector-examples/src/main/java/io/pravega/example/hadoop/wordcount/RandomTextWriter.java

```diff
@@ -193,7 +201,7 @@ public int run(String[] args) throws Exception {
 
         conf.setStrings("pravega.uri", args[1]);
         conf.setStrings("pravega.scope", args[2]);
-        conf.setStrings("pravega.stream", args[3]);
+        conf.setStrings("pravega.out.stream", args[3]);
```

Comment:
Just to check, have you verified that `WordCount` works as before after this config change?

Reply:
Yes